### DISCUSSION TITLE
How do you upgrade or install newest indy packages

### DISCUSSION THREAD

**Discussion by RogelioGaytan** [IndySockets/Indy#596] [Q&A] [answered]
Hello 

   I'm new to delphi indy library, ihave to use Delphi 2007 wich comes with indy 10.1.

   I want to upgrade to latest indy version.

How can i achieve that?

**Comment by rlebeau** [comment, marked as answer]
Download the `/Lib` source folder from Indy's GitHub repo at https://github.com/IndySockets/Indy/, and then follow the installation instructions at https://github.com/IndySockets/Indy/wiki/Updating-Indy.

**Comment by RogelioGaytan** [comment]
thank you for pointng me to the right direction
